Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md

```diff
@@ -4993,8 +5270,844 @@ Apply action to quarantined file by file IDs or filter.
+| scan_completed_on | UTC-format time of scan start to filter by. | Optional | 
+| offset | Starting index of overall result set from which to return ids. | Optional | 
+| limit | Maximum number of resources to return. | Optional | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| CrowdStrike.ODSScan.id | String | a unique identifier for the scan event. | 
+| CrowdStrike.ODSScan.cid | String | a unique identifier for the client that triggered the scan. | 
+| CrowdStrike.ODSScan.profile_id | String | a unique identifier for the scan profile used in the scan. | 
+| CrowdStrike.ODSScan.description | String | the ID of the description of the scan. | 
+| CrowdStrike.ODSScan.scan_inclusions | String | The files or folders included in the scan. | 
+| CrowdStrike.ODSScan.initiated_from | String | The source of the scan initiation. | 
+| CrowdStrike.ODSScan.quarantine | Boolean | If was set to quarantine. | 
+| CrowdStrike.ODSScan.cpu_priority | Number | The CPU priority for the scan \(1-5\). | 
+| CrowdStrike.ODSScan.preemption_priority | Number | The preemption priority for the scan. | 
+| CrowdStrike.ODSScan.metadata.host_id | String | A unique identifier for the host that was scanned. | 
+| CrowdStrike.ODSScan.metadata.host_scan_id | String | A unique identifier for the scan that was performed on the host. | 
+| CrowdStrike.ODSScan.metadata.scan_host_metadata_id | String | A unique identifier for the metadata associated with the host scan. | 
+| CrowdStrike.ODSScan.metadata.filecount.scanned | Number | The number of files that were scanned. | 
+| CrowdStrike.ODSScan.metadata.filecount.malicious | Number | The number of files that were identified as malicious. | 
+| CrowdStrike.ODSScan.metadata.filecount.quarantined | Number | The number of files that were quarantined. | 
+| CrowdStrike.ODSScan.metadata.filecount.skipped | Number | The number of files that were skipped during the scan. | 
+| CrowdStrike.ODSScan.metadata.filecount.traversed | Number | The number of files that were traversed during the scan. | 
+| CrowdStrike.ODSScan.metadata.status | String | The status of the scan on this host. \(e.q., "pending", "running", "completed", or "failed"\). | 
+| CrowdStrike.ODSScan.metadata.started_on | Date | The date and time that the scan started. | 
+| CrowdStrike.ODSScan.metadata.completed_on | Date | The date and time that the scan completed. | 
+| CrowdStrike.ODSScan.metadata.last_updated | Date | The date and time that the metadata was last updated. | 
+| CrowdStrike.ODSScan.status | String | The status of the scan. \(e.q., "pending", "running", "completed", or "failed"\). | 
+| CrowdStrike.ODSScan.hosts | String | A list of the host IDs that were scanned. | 
+| CrowdStrike.ODSScan.endpoint_notification | Boolean | A boolean value indicating whether endpoint notifications are enabled. | 
+| CrowdStrike.ODSScan.pause_duration | Number | The number of minutes to pause between scanning each file in hours. | 
```

Comment:
Not sure about this description.
Its the "number of minutes", but what does it mean "in hours"?